### DISCUSSION TITLE
我能不能请你们开始抵制小米，不停舔的结果就是现在，death by a thousand cuts

### DISCUSSION THREAD

**Discussion by Originalimoc** [MlgmXyysd/Xiaomi-BootLoader-Questionnaire#167] [General]
毕竟Louis-the-ranting-artist说得好，见血，牌子都是虚的把自己当大爷的gǔn得越早越好：https://youtu.be/Ahw1cppZi-g
```
1561580 有部分用户填空题的答案雷同，甚至标点符号、格式、措辞几乎完全一致，导致其答题无效，请你谈谈对于这类违规用户应该如何处理 【填空题】
0
✔️ 至少需要输入 20 个字符。「说实在的，米米你要是不想给解锁可以直说，犯不着出这种题来恶心人。用户是你爹不是你孙子，社区也不是给你养土皇帝的地方，某些员工手里有了点权力，就忘了究竟是谁成就了小米的今天。」
1557482 请写出您申请BL解锁的具体用途 【填空题】
0
✔️ 至少需要输入 20 个字符。「我的手机我做主，只要机器是我从正规途径合法购得的，你管我解锁用来干嘛？」
```
呵。
SB玩意

**Comment by Originalimoc** [comment]
没什么必须是他们才能提供的，小米已死，华硕已死。都不认清现实。一加exists, Xperia exists, Pixel exists, even Samsung or Nothing exists... 等我手里ROG和11TPro退休了就换Xperia或Pixel和一加。再不抵制全部都是苹果。

**Reply by Payton9000** [reply]
是的，当场买小米就是看在能解bl以及玩机资源多上，现在看来还支持发烧友的只有一加了

**Reply by Originalimoc** [reply]
> 是的，当场买小米就是看在能解bl以及玩机资源多上，现在看来还支持发烧友的只有一加了

@Payton9000  好像现在realme也可以解，不过我没试过

**Reply by dixtdf** [reply]
听说一加后续也要加强bl解锁管理了，难道以后只能买欧版三星了？

**Reply by dixtdf** [reply]
反正我现在用13pm，搞小米解锁就是凑个热闹，没想到这么难，以后不放开的情况下不会考虑小米了

**Reply by imzhengwei** [reply]
买手机，首认苹果，次任Pixel、Nothing。次之三星，其它都靠边站。

**Comment by AlaskaSier** [comment]
米粉已轉路人、手上這台是最後一台小米

**Comment by pierrecashon** [comment]
买了小米14pro才发现不能解锁，小米就是纯纯犯贱恶心人，耍猴

**Reply by FenghaoZhu** [reply]
目前普通人只能付费用xiaomikey解锁
其他路基本都堵死了

**Reply by Originalimoc** [reply]
Or, sell it. Take the L, say f off.

**Reply by qingc1237** [reply]
xiaomikey怎么用？听他们说涨价了

**Reply by Aa244750146** [reply]
现在已经100刀了

**Reply by FenghaoZhu** [reply]
100刀？？？
这已经不是抢钱了
这个是抄家啊啊啊啊

**Reply by Aa244750146** [reply]
或者鱼上面买资格，大概2张到3张左右

**Comment by MlgmXyysd** [comment]
事实是米猴还不少，可能这就是小米敢锁bl的资本吧，毕竟不管干了什么都有人会去洗地

**Reply by frlda** [reply]
还真是，话术一致说解锁有什么用，澎湃2已经很流畅，成为安卓标杆了

**Comment by xqneko** [comment]
我功课没做足，还偏偏买了Redmi Note 13，解锁不了才来搜，就晚了……还好是以低成本试探了一下小米产品，但是哪知道小米搞这出呢

**Reply by xbn2002** [reply]
note 13 是不管你怎么样都解不了

**Reply by xbn2002** [reply]
就没有官方的解锁渠道

**Reply by Originalimoc** [reply]
Mi: 
![w700d1q75cms](https://github.com/MlgmXyysd/Xiaomi-BootLoader-Questionnaire/assets/20218140/c3190cfa-cce0-4b34-bd4d-698368c65095)

**Comment by 501658362** [comment]
粉转黑了

**Comment by dixtdf** [comment]
我目前知道的只有欧版三星比较好解，还有其他比较好解的吗，一加也要收紧了不考虑

**Reply by imzhengwei** [reply]
Pixel和Nothing

**Comment by PoliteYoung** [comment]
大早上告诉我结束了，太会了

**Comment by Enaium** [comment]
下部手机不买小米了

**Comment by YujiaCheng1996** [comment]
我去年暑假刚买14Utra就关闭解锁通道了，今年再开我只考了50分，我的下部手机肯定是不考试能解锁的，是不是小米看造化了。

**Comment by Diffused7245** [comment]
一加玩机资源开始变多了，好事。
现在小米还是靠着以前的惯性向前运动，但后面资源肯定越来越少，国外不用小米高考，但需要北京时间0点抢名额，一天据说就20个还是几百个来着。

**Comment by Enaium** [comment]
![Screenshot_2026-01-28-19-56-06-70_fc704e6b13c4fb26bf5e411f75da84f2.jpg](https://github.com/user-attachments/assets/077984f9-d040-45a9-b80f-35ed2fc353ad)

换1+了
